manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc

```diff
@@ -22,58 +22,58 @@ There are two main steps in preparing your migration to Apache Unomi 2.0:
 
 Note that it is currently not possible to modify or surcharge an existing system-deployed JSON schema via the REST API. It is however possible to deploy new schemas and manage them through the REST API on the `/cxs/jsonSchema` endpoint.
-If you are currently using custom properties on an Apache Unomi-provided event type, 
+If you are currently using custom properties on an Apache Unomi-provided event type,
 you will need to either change to use a new custom eventType and create the corresponding schema or to create a Unomi schema extension. You can find more details in the <<JSON schemas,JSON Schema>> section of this documentation.
 
-You can use, as a source of inspiration for creating new schemas, Apache Unomi 2.0 schema located at: 
+You can use, as a source of inspiration for creating new schemas, Apache Unomi 2.0 schema located at:
  https://github.com/apache/unomi/tree/master/extensions/json-schema/services/src/main/resources/META-INF/cxs/schemas[extensions/json-schema/services/src/main/resources/META-INF/cxs/schemas].
 
-Finally, and although it is technically feasible, we recommend against creating permissive JSON Schemas allowing any event payload. This requires making sure that you don't allow open properties by using JSON schema keywords such as https://json-schema.org/understanding-json-schema/reference/object.html#unevaluated-properties[unevaluated properties]
+Finally, and although it is technically feasible, we recommend against creating permissive JSON Schemas allowing any event payload. This requires making sure that you don't allow undeclared properties by setting JSON schema keywords such as https://json-schema.org/understanding-json-schema/reference/object.html#unevaluated-properties[unevaluated properties] to `false`.
 
 === Migrating your existing data
 
 ==== Elasticsearch version and capacity
 
-While still using Unomi 1.6, the first step will be to upgrade your Elasticsearch to 7.17.5. 
+While still using Unomi 1.6, the first step will be to upgrade your Elasticsearch to 7.17.5.
 Documentation is available on https://www.elastic.co/guide/en/elasticsearch/reference/7.17/setup-upgrade.html[Elasticsearch's website].
 
-Your Elasticsearch cluster must have enough capacity to handle the migration. 
-At a minimum, the required capacity must be greater than the size of the dataset in production + the size of the largest index.
+Your Elasticsearch cluster must have enough capacity to handle the migration.
+At a minimum, the required capacity storage capacity must be greater than the size of the dataset in production + the size of the largest index and any other settings should at least be as big as the source setup (preferably higher).
```

Comment:
I would use only + or only "and" as the reader is expected to calculate a sum

Reply:
I fixed this by breaking to another sentence.